docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -7,6 +7,35 @@ This topic covers common troubleshooting issues when using {elastic-sec} <<sec-m
 [[ts-endpoints]]
 === Endpoints
 
+[discrete]
+[[ts-unhealthy-agent]]
+.Unhealthy {agent} status
+[%collapsible]
+====
+In some cases, an `Unhealthy` {agent} status may be caused by a failure in the {endpoint-cloud-sec} integration policy. In this situation, the integration and any failing features are flagged on the agent details page in {fleet}. Expand each section and subsection to display individual responses from the agent.
+
+TIP: Integration policy response information is also available from the *Endpoints* page in the {security-app} (*Manage* -> *Endpoints*, then click the link in the *Policy status* column).
+
+[role="screenshot"]
+image::images/unhealthy-agent-fleet.png[Agent details page in {fleet} with Unhealthy status and integration failures]
+
+Common causes of failure in the {endpoint-cloud-sec} integration policy include missing prerequisites or unexpected system configuration. Consult the following topics to resolve a specific error:
+
+- <<system-extension-endpoint,Approve the system extension for {elastic-endpoint}>> (macOS)
+- <<enable-fda-endpoint,Enable Full Disk Access for {elastic-endpoint}>> (macOS)
+- <<linux-deadlock,Resolve a potential system deadlock>> (Linux)
+
+TIP: If the {endpoint-cloud-sec} integration policy is not the cause of the `Unhealthy` agent status, refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} troubleshooting] for help with the {agent}.
+====
+
+[discrete]
+[[linux-deadlock]]
+.Disabled to avoid potential system deadlock (Linux)
+[%collapsible]
+====
+This section is a placeholder for future documentation.
```

Comment:
I remember discussing this last week, but still flagging this section anyways in case you wanted to remove it. If this is intentional, disregard this comment.

Reply:
Yep, we have to have a section for this to avoid breaking other things, and content will be added soon. Leaving the placeholder instead of a blank section, which might look like an error.